examples/step-72/step-72.cc

```diff
@@ -489,40 +505,47 @@ namespace Step72
       // To do this, we need to specify how many independent variables and
       // dependent variables there are. The independent variables will be the
       // number of local degrees of freedom that our solution vector has,
-      // i.e., the number $I$ in the per-element representation of the
-      // discretized solution vector 
-      // $u_{e} (\mathbf{X}) = \sum\limits_{I} u_{e}^{I} N^{I}(\mathbf{X})$
+      // i.e., the number $j$ in the per-element representation of the
+      // discretized solution vector
+      // $u (\mathbf{x})|_K = \sum\limits_{j} U^K_i \varphi_j(\mathbf{x})$
       // that indicates how many solution coefficients are associated with
-      // each finite element.
-      // The number of dependent variables will be the number of entries in
-      // the local residual vector that we will be forming. In this particular
-      // problem (like many others that employ the 
-      // [standard Galerkin method](https://en.wikipedia.org/wiki/Galerkin_method))
-      // the number of local solution coefficients matches the number of local
-      // residual equations.
+      // each finite element. In deal.II, this equals
+      // FiniteElement::dofs_per_cell. The number of dependent variables will be
+      // the number of entries in the local residual vector that we will be
+      // forming. In this particular problem (like many others that employ the
+      // [standard Galerkin
+      // method](https://en.wikipedia.org/wiki/Galerkin_method)) the number of
+      // local solution coefficients matches the number of local residual
+      // equations.
       const unsigned int n_independent_variables = local_dof_indices.size();
       const unsigned int n_dependent_variables   = dofs_per_cell;
       ADHelper ad_helper(n_independent_variables, n_dependent_variables);
 
-      // Next we inform the helper of the values of the solution about which we
-      // wish to linearize. As this is done on each element individually, we have
-      // to extract the solution coefficients from the global solution vector.
-      ad_helper.register_dof_values(current_solution, local_dof_indices);
-
+      // Next we inform the helper of the values of the solution, i.e., the
+      // actual values for $U_j$ about which we
+      // wish to linearize. As this is done on each element individually, we
+      // have to extract the solution coefficients from the global solution
+      // vector. In other words, we define all of those coefficients $U_j$
+      // where $j$ is a local degree of freedom as the independent variables
+      // that enter the computation of the vector $F(U)$ (the dependent
```

Comment:
```suggestion
      // that enter the computation of the vector $F(U)^{K}$ (the dependent
```
? Or do you think that this is sufficiently well implied by the phrases "As this is done on each element individually" and "local degree of freedom"?